src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -482,6 +482,11 @@ Sie können die Hintergrundaktualisierung für die App jederzeit in den Gerätee
 "ExposureSubmissionResult_RegistrationDate" = "Registriert am";
 
 /* Exposure Submission */
+
+"ExposureSubmissionError_InvalidPayloadOrHeaders" = "Invalide Payload oder Header erhalten.";
+
+"ExposureSubmissionError_RequestCouldNotBeBuilt" = "Die Submission Request konnte nicht korrekt gebaut werden.";
```

Comment:
are those the official translations?

Reply:
Nope.  I will ask @kaluju whether there are some english/german translations for these error messages.